src/plugins/chart_expressions/expression_metric/common/constants.ts

```diff
@@ -15,3 +15,23 @@ export const LabelPosition = {
   BOTTOM: 'bottom',
   TOP: 'top',
 } as const;
+
+export const AvailableMetricIcons = {
```

Comment:
@gvnmagni @markov00 can you help me with the icons dropdown for the metric? I added the sortUp/Down and compute as they are also on the EC documentation and some others that also exist in annotations. But I am not sure if they are all helpful for the metric. Wdyt?

Reply:
I was about to ask if we could add all icons from EUI but I realized how many are they, should we make a selection? How can we define the most important ones?

Reply:
Probably better to use "pin" instead of "pinFilled" so that we are more consistent with other icons that are not filled, for the same reason I think we can avoid including "starFilled" if there are no other important reasons to have it since we already have "starEmpty"

From EUI icons, I would add:
- "globe", to have something related to geography, I'm thinking about having Metric related to countries or similar
- "temperature", to allow machines monitoring

From my point of view I don't see any other icons that might be important to have :)

Reply:
Thanx Gio, done https://github.com/elastic/kibana/pull/154210/commits/86819568987e629e067e1a9ccb22eac88d6b4594